<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vantanaderivacion</class>
 <widget class="QDialog" name="vantanaderivacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Método derivacion</string>
  </property>
  <widget class="QLineEdit" name="numero_iteraciones">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Iteraciones:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ecuacion">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ecuación:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valorX">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valor de x:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valorH">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valor de h:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="mitabla">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>921</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>130</number>
   </attribute>
   <column>
    <property name="text">
     <string>h</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Xi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Xi+1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>f(Xi)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>f(Xi+1)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Aproximación</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor Real</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Error</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="boton_aceptar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>227</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
